service/controller/v11/resource/endpoint/create.go

```diff
@@ -0,0 +1,67 @@
+package endpoint
```

Comment:
All this is the endpoint resource from the endpoint-operator. Nothing changed on the logic. I only modified some logging and such to align it with the kvm-operator internals.